memcache/memcache.go

```diff
@@ -379,8 +383,13 @@ func (c *Client) withKeyRw(key string, fn func(*bufio.ReadWriter) error) error {
 }
 
 func (c *Client) getFromAddr(addr net.Addr, keys []string, cb func(*Item)) error {
+	cmd := "gets"
+	if c.DisableCAS {
```

Comment:
it looks like the meat of this PR is here... the DisableCAS bool that changes the command sent.